src/components/quality-page/quality-content-group.styles.scss

```diff
@@ -0,0 +1,38 @@
+.top-section {
+    display: flex;
+}
+
+.number {
+    font-size: 5.5em;
+    font-family: 'Times New Roman', Times, serif;
+    font-style: italic;
+    color: #BFB4AD;
+}
+
+.subheading {
+    font-size: 1.75em;
+    font-family: YuMincho;
+    font-style: italic;
+    color: #5A4537;
+    width: 22vw;
+    line-height: 6.6vh;
+    margin-top: 0.6vh;
+    margin-left: 2.3vw;
+}
+
+.line {
+    border-top: 2px solid #BFB4AD;
+    width: 31vw;
+    margin-top: 1vw;
+    margin-bottom: 1.5vw;
+}
+
+.textbox {
```

Comment:
textbox is super generic so it should be placed in app.css and used for all textboxes.
or make the name specific to what its addressing

Reply:
Changed the textbox to a more specific name